src/index.html

```diff
@@ -1,13 +1,257 @@
+      </section>
+
+      <section class="categories conteiner">
+        <h2 class="section-title section-title--inline">Sound. Vision.<br>It’s your pick.</h2>
+
+        <div class="categories__list">
+
+          <article class="category">
+            <a href="#portable-speakers" class="category__link">
+              <img
+                src="./img/products/portable speakers1.png"
+                alt="portable speakers prewiev"
+                class="category__image">
+            </a>
+            <a href="#portable-speakers" class="category__link">
+              <img
+                src="./img/products/portable speakers2.png"
+                alt="portable speakers prewiev"
+                class="category__image">
+            </a>
+            <h3 class="category__title">Portable speakers</h3>
+          </article>
+
+          <article class="category">
+            <a href="#earphones" class="category__link">
+              <img
+                src="./img/products/earphones1.png"
+                alt="earphones prewiev"
+                class="category__image">
+            </a>
+            <a href="#earphones" class="category__link">
+              <img
+                src="./img/products/earphones2.png"
+                alt="earphones prewiev"
+                class="category__image">
+            </a>
+            <h3 class="category__title">Earphones</h3>
+          </article>
+
+          <article class="category">
+            <a href="#smart-home" class="category__link">
+              <img
+                src="./img/products/smart home1.png"
+                alt="smart home prewiev"
+                class="category__image">
+            </a>
+            <a href="#smart-home" class="category__link">
+              <img
+                src="./img/products/smart home2.png"
+                alt="smart home prewiev"
+                class="category__image">
+            </a>
+            <h3 class="category__title">Smart Home</h3>
+          </article>
+
+          <a href="#categories__all" class="button categories__button">
+            View all
+          </a>
+        </div>
+      </section>
+
+      <section class="about">
+        <img
+          src="./img/bg-page.png"
+          alt="timeless-bg image"
+          class="about__bg-image">
+
+        <div class="about__content">
+          <div class="conteiner about__content--grid">
+            <h2 class="section-title section-title--block">Timeless, for 50 years and counting</h2>
+            <p class="about__description">Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="contact conteiner">
+        <h2 class="section-title section-title--inline">Contact us</h2>
+
+        <form
```

Comment:
have you tried it (the [checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md))?
![image](https://github.com/mate-academy/layout_landing-page/assets/86923089/823f7e7a-0640-4882-8bb8-ac76186482ae)